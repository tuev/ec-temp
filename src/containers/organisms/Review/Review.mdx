# Stories mdx for Review

import { Meta, Story } from '@storybook/addon-docs/blocks'
import * as stories from './Review.stories.tsx'
import Review from './Review.tsx'

<Story name="basic" />