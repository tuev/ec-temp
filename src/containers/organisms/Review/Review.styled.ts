/**
 *
 * Styled for Review
 *
 */

import styled from 'styled-components'
import { space } from 'styled-system'
import { COLOR } from '../../../theme/colors'
import Divider from '@material-ui/core/Divider'

export const ReviewContainer = styled.div`
  ${space}
  background-color: ${COLOR.gray2};
  padding-top: 40px;
  padding-bottom: 40px;
`
export const ReviewDivider = styled(Divider)`
  margin-bottom: 45px;
  color: ${COLOR.gray};
`
