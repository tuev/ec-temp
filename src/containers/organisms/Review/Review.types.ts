import { SpaceProps } from 'styled-system'
import { IReviewList } from '../../../components/molecules/ReviewList/ReviewList.types'

export type BaseReviewProps = IReviewList & SpaceProps
