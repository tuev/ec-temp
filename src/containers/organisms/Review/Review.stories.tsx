/**
 *
 *  Stories for Review
 *
 */

import React from 'react'
import { action } from '@storybook/addon-actions'
import Review from './Review'
import { reviewList } from '../../../components/molecules/ReviewList/mock'

export default {
  component: Review,
  includeStories: ['basic'],
  title: 'Review',
}

export const basic = () => (
  <div>
    <Review list={reviewList} />
  </div>
)
