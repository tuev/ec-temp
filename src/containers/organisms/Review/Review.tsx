/**
 *
 * Review
 *
 */

import React, { FC } from 'react'
import { ReviewForm, ReviewList } from '../../../components/molecules'
import { BaseReviewProps } from './Review.types'
import { ReviewContainer, ReviewDivider } from './Review.styled'
import { StylesProvider } from '@material-ui/core/styles'

const Review: FC<BaseReviewProps> = props => {
  const { list } = props
  return (
    <ReviewContainer>
      <ReviewList list={list} />
      <StylesProvider injectFirst={true}>
        <ReviewDivider />
      </StylesProvider>
      <ReviewForm />
    </ReviewContainer>
  )
}

export default Review
