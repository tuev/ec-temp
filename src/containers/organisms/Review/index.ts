/**
 *
 * Index for Review
 *
 */

import Review from './Review'

export default Review
