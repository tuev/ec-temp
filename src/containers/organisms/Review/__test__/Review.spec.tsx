/**
 *
 * Tests for Review
 *
 */

import React from 'react'

import { render, fireEvent } from '@testing-library/react'
import Review from '..'

describe('App Review', () => {
  it('Test get default Review no params ', () => {
    // test snapshot
    const wrapper = render(<Review />)
    expect(wrapper).toMatchSnapshot()
  })
})
