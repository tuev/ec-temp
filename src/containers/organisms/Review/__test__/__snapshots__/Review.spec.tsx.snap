// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Review Test get default Review no params  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c6 {
  min-width: 163px !important;
  min-height: 48px !important;
  font-size: 14px;
  border-radius: 32px;
}

.c1 {
  margin-bottom: 32px;
  font-size: 24px;
  line-height: 1.5;
  font-family: 'Montserrat Alternates';
  font-weight: bold;
  color: #2E2E2E;
}

.c3 {
  font-size: 16px;
  line-height: 1.5;
  font-family: 'Open Sans';
  color: #2E2E2E;
}

.c5 {
  padding-top: 10px;
  font-size: 16px;
  line-height: 1.5;
  font-family: 'Open Sans';
  color: #2E2E2E;
}

.c4 {
  font-weight: 600;
}

.c0 {
  background-color: #F2F2F2;
  padding-top: 40px;
  padding-bottom: 40px;
}

.c2 {
  margin-bottom: 45px;
  color: #F7F7F7;
}

<body>
    <div>
      <div
        class="c0"
      >
        <div
          class=""
        >
          <p
            class="MuiTypography-root c1 MuiTypography-body1"
            customvariant="header2"
            mb="4"
          >
            Product reviews
          </p>
        </div>
        <hr
          class="MuiDivider-root c2"
        />
        <div>
          <p
            class="MuiTypography-root c1 MuiTypography-body1"
            customvariant="header2"
            mb="4"
          >
            Add a review
          </p>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
              >
                <p
                  class="MuiTypography-root c3 c4 MuiTypography-body1"
                  customvariant="body1"
                >
                  Choose a nickname
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root "
                >
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input"
                      name="nickname"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="PrivateNotchedOutline-root-180 MuiOutlinedInput-notchedOutline"
                      style="padding-left: 8px;"
                    >
                      <legend
                        class="PrivateNotchedOutline-legend-181"
                        style="width: 0.01px;"
                      >
                        <span>
                          ​
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
              >
                <p
                  class="MuiTypography-root c5 c4 MuiTypography-body1"
                  customvariant="body1"
                  pt="10"
                >
                  Your review
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                >
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
                  >
                    <textarea
                      aria-invalid="false"
                      class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
                      name="reviewText"
                      rows="4"
                    />
                    <fieldset
                      aria-hidden="true"
                      class="PrivateNotchedOutline-root-224 MuiOutlinedInput-notchedOutline"
                      style="padding-left: 8px;"
                    >
                      <legend
                        class="PrivateNotchedOutline-legend-225"
                        style="width: 0.01px;"
                      >
                        <span>
                          ​
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
              >
                <p
                  class="MuiTypography-root c3 c4 MuiTypography-body1"
                  customvariant="body1"
                >
                  Overall rating
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
              >
                <span
                  class="MuiRating-root "
                >
                  <label
                    class="MuiRating-label"
                    for="app-rating-1"
                  >
                    <span
                      class="MuiRating-icon MuiRating-iconEmpty"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiRating-visuallyhidden"
                    >
                      1 Star
                    </span>
                  </label>
                  <input
                    class="MuiRating-visuallyhidden"
                    id="app-rating-1"
                    name="app-rating"
                    type="radio"
                    value="1"
                  />
                  <label
                    class="MuiRating-label"
                    for="app-rating-2"
                  >
                    <span
                      class="MuiRating-icon MuiRating-iconEmpty"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiRating-visuallyhidden"
                    >
                      2 Stars
                    </span>
                  </label>
                  <input
                    class="MuiRating-visuallyhidden"
                    id="app-rating-2"
                    name="app-rating"
                    type="radio"
                    value="2"
                  />
                  <label
                    class="MuiRating-label"
                    for="app-rating-3"
                  >
                    <span
                      class="MuiRating-icon MuiRating-iconEmpty"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiRating-visuallyhidden"
                    >
                      3 Stars
                    </span>
                  </label>
                  <input
                    class="MuiRating-visuallyhidden"
                    id="app-rating-3"
                    name="app-rating"
                    type="radio"
                    value="3"
                  />
                  <label
                    class="MuiRating-label"
                    for="app-rating-4"
                  >
                    <span
                      class="MuiRating-icon MuiRating-iconEmpty"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiRating-visuallyhidden"
                    >
                      4 Stars
                    </span>
                  </label>
                  <input
                    class="MuiRating-visuallyhidden"
                    id="app-rating-4"
                    name="app-rating"
                    type="radio"
                    value="4"
                  />
                  <label
                    class="MuiRating-label"
                    for="app-rating-5"
                  >
                    <span
                      class="MuiRating-icon MuiRating-iconEmpty"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                        />
                      </svg>
                    </span>
                    <span
                      class="MuiRating-visuallyhidden"
                    >
                      5 Stars
                    </span>
                  </label>
                  <input
                    class="MuiRating-visuallyhidden"
                    id="app-rating-5"
                    name="app-rating"
                    type="radio"
                    value="5"
                  />
                </span>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained c6 MuiButton-containedPrimary Mui-disabled Mui-disabled"
                  disabled=""
                  tabindex="-1"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    Submit
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-fzXfLT crLWPT"
    >
      <div
        class="sc-fzXfLR dNpFBq"
      >
        <p
          class="MuiTypography-root sc-AykKJ khvmKv sc-AykKK gcTcIC MuiTypography-body1"
          customvariant="header2"
          mb="4"
        >
          Product reviews
        </p>
      </div>
      <hr
        class="MuiDivider-root sc-fzXfLU gDXfkL"
      />
      <div>
        <p
          class="MuiTypography-root sc-AykKJ khvmKv sc-AykKK gcTcIC MuiTypography-body1"
          customvariant="header2"
          mb="4"
        >
          Add a review
        </p>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
            >
              <p
                class="MuiTypography-root sc-AykKJ iexqoL sc-fzXfLO jEaXPj MuiTypography-body1"
                customvariant="body1"
              >
                Choose a nickname
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
            >
              <div
                class="MuiFormControl-root MuiTextField-root sc-AykKE fbCwcn"
              >
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input"
                    name="nickname"
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-180 MuiOutlinedInput-notchedOutline"
                    style="padding-left: 8px;"
                  >
                    <legend
                      class="PrivateNotchedOutline-legend-181"
                      style="width: 0.01px;"
                    >
                      <span>
                        ​
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
            >
              <p
                class="MuiTypography-root sc-AykKJ kxCBjo sc-fzXfLO jEaXPj MuiTypography-body1"
                customvariant="body1"
                pt="10"
              >
                Your review
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
            >
              <div
                class="MuiFormControl-root MuiTextField-root sc-AykKF hjTiPI MuiFormControl-fullWidth"
              >
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
                >
                  <textarea
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
                    name="reviewText"
                    rows="4"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-224 MuiOutlinedInput-notchedOutline"
                    style="padding-left: 8px;"
                  >
                    <legend
                      class="PrivateNotchedOutline-legend-225"
                      style="width: 0.01px;"
                    >
                      <span>
                        ​
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
            >
              <p
                class="MuiTypography-root sc-AykKJ iexqoL sc-fzXfLO jEaXPj MuiTypography-body1"
                customvariant="body1"
              >
                Overall rating
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
            >
              <span
                class="MuiRating-root sc-AykKH leYRC"
              >
                <label
                  class="MuiRating-label"
                  for="app-rating-1"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiRating-visuallyhidden"
                  >
                    1 Star
                  </span>
                </label>
                <input
                  class="MuiRating-visuallyhidden"
                  id="app-rating-1"
                  name="app-rating"
                  type="radio"
                  value="1"
                />
                <label
                  class="MuiRating-label"
                  for="app-rating-2"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiRating-visuallyhidden"
                  >
                    2 Stars
                  </span>
                </label>
                <input
                  class="MuiRating-visuallyhidden"
                  id="app-rating-2"
                  name="app-rating"
                  type="radio"
                  value="2"
                />
                <label
                  class="MuiRating-label"
                  for="app-rating-3"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiRating-visuallyhidden"
                  >
                    3 Stars
                  </span>
                </label>
                <input
                  class="MuiRating-visuallyhidden"
                  id="app-rating-3"
                  name="app-rating"
                  type="radio"
                  value="3"
                />
                <label
                  class="MuiRating-label"
                  for="app-rating-4"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiRating-visuallyhidden"
                  >
                    4 Stars
                  </span>
                </label>
                <input
                  class="MuiRating-visuallyhidden"
                  id="app-rating-4"
                  name="app-rating"
                  type="radio"
                  value="4"
                />
                <label
                  class="MuiRating-label"
                  for="app-rating-5"
                >
                  <span
                    class="MuiRating-icon MuiRating-iconEmpty"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiRating-visuallyhidden"
                  >
                    5 Stars
                  </span>
                </label>
                <input
                  class="MuiRating-visuallyhidden"
                  id="app-rating-5"
                  name="app-rating"
                  type="radio"
                  value="5"
                />
              </span>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-flex-end"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-9"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained sc-AykKI evYzjc MuiButton-containedPrimary Mui-disabled Mui-disabled"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Submit
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
